app/controllers/calls_controller.rb

```diff
@@ -10,20 +17,133 @@ def begin
     end
     render xml: response.to_s
   end
-    
+
   def dial
-    dial_number = params['Digits']
+    boot_twilio
+    @@dial_number = params['Digits']
+    @call = @@client.calls.create(
+      url: "http://c0ae5a76.ngrok.io/calls/answered",
+      to: @@dial_number,
+      from: @@user_number
+    )
+    #user = @@client.conferences('conference').participants(@@user_callSid).create(from: @@user_number, to: @@dial_number)
+    response = Twilio::TwiML::VoiceResponse.new do |r|
+      r.say(message: 'Forwarding your call now to' + @@dial_number)
+      r.say(message: 'If you would like us to wait for you. Please press star 0 0')
+      r.dial(hangupOnStar: 'true') do |d|
+        d.conference('conference', muted:'False', beep:'False',statusCallbackEvent:'join leave', statusCallback:'/calls/conference',
+          statusCallbackMethod:'POST')
+        end
+      r.gather(action: '/calls/confirmwait', method: 'POST', numdigits: 2)
+      r.redirect('/calls/rejoinconference')
+    end
+    render xml: response.to_s
+  end
+
+  def answered
+    @@business_callSid = params['CallSid']
+    puts 'business callsid ' + @@business_callSid
+    response = Twilio::TwiML::VoiceResponse.new do |r|
+      r.say(message: 'you answered the call')
+      r.dial do |d|
+        d.conference('conference', muted:'False', beep:'False',statusCallbackEvent:'join leave', statusCallback:'/calls/conference',
+          statusCallbackMethod:'POST')
+        end
+    end
+    render xml: response.to_s
+  end
+
+  def conference
+    @event = params["StatusCallbackEvent"]
```

Comment:
Does this need to be an instance variable? Usually instance variables in a Rails controller are used for values that need to be shared to the view.